liquibase-core/src/main/java/liquibase/datatype/core/BigIntType.java

```diff
@@ -70,6 +70,9 @@ public DatabaseDataType toDatabaseDataType(Database database) {
                 }
             }
         }
+        if (database instanceof SybaseDatabase) {
```

Comment:
Can the SybaseASADatabase and SybaseDatabase "ifs" be included as an "or" in the "if" at line 56?

Reply:
We were actually already checking for sybase up on line 44 so I just moved SybaseASA up there. 

It doesn't super matter, I had just left it as it came though in the merge